Shared/Managers/KeyboardManager.swift

```diff
@@ -37,17 +37,15 @@ class KeyboardManager {
         case none // No event has happened
     }
 
-    deinit {
-        self.cancellables.forEach { cancellable in
-            cancellable.cancel()
-        }
-    }
-
     func reset() {
         self.cachedKeyboardEndFrame = .zero
         self.inputAccessoryView = nil
         self.willKeyboardShow = false
         self.isKeyboardShowing = false
+        self.cancellables.forEach { cancellable in
+            cancellable.cancel()
+        }
+        self.cancellables.removeAll()
```

Comment:
Deinit will never get called on a static instance. This was causing significant memory leaks.

Reply:
Bing bong